frontend/scripts/setup/ubuntu_setup.sh

```
@@ -0,0 +1,266 @@
+  setup_android_sdk
+
+  echo_info "Configuring Flutter channels and platforms..."
+  flutter config --no-enable-linux-desktop
+  flutter config --enable-web
+  flutter config --enable-android
+
+  echo_info "Running Flutter Doctor..."
+  flutter doctor -v || echo_error "Flutter Doctor found issues. Please review the output above."
+
+  echo_success "Flutter development environment setup completed!"
+  source ~/.bashrc
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)